.github/workflows/test.yml

```diff
@@ -46,6 +46,9 @@ jobs:
           - none
         database:
           - internal
+        iop:
+          - enabled
+          - disabled
```

Comment:
This will run IoP enabled tests for *every* combination, thus doubling the matrix. Do we *need* that?

See how we used `security` and `database` with a single entry in the matrix itself and then special additions via `include` (resulting in one additional job)

Reply:
Want an explanation?